src/frontend/jsonapi/basics.md

```diff
@@ -482,7 +517,7 @@ To minimize the number of requests, the Aimeos JSON API can add related resource
     });
     ```
 
-This returns the texts associated with the products within the same request:
+This returns the product(s) as well as the text(s) associated with the product(s) within the same request:
```

Comment:
Would prefer somehing like this: "This returns the product or products as well as the texts associated with the products within the same request". The text items are always a list even if it only contains one item.